web/server/codechecker_server/server.py

```diff
@@ -200,17 +200,30 @@ def end_headers(self):
 
         SimpleHTTPRequestHandler.end_headers(self)
 
+    @staticmethod
+    def _get_client_host_port(address):
+        """
+        Returns the host and port of the request's address, and whether it
+        was an IPv6 address.
+        """
+        if len(address) == 2:
+            return address[0], address[1], False
```

Comment:
Could be the [ipaddress](https://docs.python.org/3/library/ipaddress.html#module-ipaddress) module used here to store the ip addresses?
It could parse the address into IPv4Address or IPv6Address objects, both objects have a version attribute which could simplify the current logic a bit.
What do you think?

Reply:
This module seems way too loaded for this purpose. The only reason we need this distinction is to properly pretty-print an IPv6 address in the log.